vue/configurator/src/components/Configurator.vue

```diff
@@ -68,6 +68,72 @@
               v-model="prop.value"
               size="large"
             />
+            <!-- data-media-background has a type of JSON string -->
+            <el-row
+              v-else-if="prop.type === WidgetPropType.INTERFACE"
+              style="
+                align-items: center;
+                border-color: lightgrey;
+                border-style: solid;
+                border-width: thin;
+                border-radius: 5px;
+                height: 150px;
+                width: 250px;
+              "
+            >
+              <el-row v-for="(config, key) in prop.value" :key="key">
+                <!-- config needs to be confirmed as an object for it to have properties like 'type' since
+                    it can be a string or boolean as defined in WidgetProps.ts -->
+                <!-- Placeholder uses "key" instead of config.name because config can be string | boolean | WidgetPropDefiniton -->
+                <el-input
+                  v-if="
+                    typeof config === 'object' &&
+                    config?.type === WidgetPropType.STRING
+                  "
+                  v-model="config.value"
+                  class="w-50 m-2"
+                  :placeholder="key"
+                />
+                <div
+                  v-else-if="
+                    typeof config === 'object' &&
+                    config?.type === WidgetPropType.ARRAY
+                  "
+                >
+                  <label id="multipleInputs"> colors example: red, blue </label>
```

Comment:
just caught this this time around -- if there are more config props with type = `ARRAY`, you would have multiple labels with the same id + placeholder test. that wouldn't be good for accessibility issues.

Reply:
true! I actually decided I didn't need to ID this label since any labels created by config props with type = `ARRAY` would then be styled using the selector `label` rather than creating accessibility issues by potentially using same `ids`